src/adzerk/response_models.rs

```diff
@@ -315,10 +323,10 @@ mod tests {
                     }
                     5 => {
                         decision.contents[0].body = Some(
-                            r#"{\
-                                "topic_arts_and_entertainment":"",\
-                                "topic_autos_and_vehicles":"true",\
-                                "topic_beauty_and_fitness":"true"\
+                            r#"{
```

Comment:
Rust string literals use `\` to escape newline characters and the whitespace at the beginning of the next line. Apparently, this feature isn't avaiable in raw strings, so the back slashes ended up in the actual string, causing JSON errors.